src/receive_imf.rs

```diff
@@ -1167,7 +1163,7 @@ async fn add_parts(
             if chat_id_blocked != Blocked::Not {
                 if let Some(chat_id) = chat_id {
                     chat_id.unblock_ex(context, Nosync).await?;
-                    chat_id_blocked = Blocked::Not;
+                    // Not assigning `chat_id_blocked = Blocked::Not` to avoid unused_assignments warning.
```

Comment:
Btw, can we write e.g. `drop(chat_id_blocked);`? So that if it's used later, the compiler complains